bot/exts/halloween/spooknamerate.py

```diff
@@ -356,22 +411,27 @@ async def get_responses_list(self, final: bool = False) -> Embed:
     async def get_channel(self) -> Union[TextChannel, None]:
         """Gets the sir-lancebot-channel after waiting until ready."""
         await self.bot.wait_until_ready()
-        channel = self.bot.get_channel(Channels.community_bot_commands) or await self.bot.fetch_channel(
+        channel = self.bot.get_channel(
             Channels.community_bot_commands
-        )
+        ) or await self.bot.fetch_channel(Channels.community_bot_commands)
         if not channel:
-            logger.warning("Bot is unable to get the #seasonalbot-commands channel. Please check the channel ID.")
+            logger.warning(
+                "Bot is unable to get the #seasonalbot-commands channel. Please check the channel ID."
+            )
         return channel
 
     @staticmethod
     def load_json(file: Path) -> Dict[str, str]:
         """Loads a JSON file and returns its contents."""
-        with file.open('r', encoding='utf-8') as f:
+        with file.open("r", encoding="utf-8") as f:
             return json.load(f)
 
     @staticmethod
     def in_allowed_month() -> bool:
         """Returns whether running in the limited month."""
+        if SpookNameRate.debug:
```

Comment:
`SpookNameRate.debug` is not the same as `self.debug`. One is attached to the cog instance, the other the class itself. If you need to use the static version, set on the class, then you need to use it everywhere else.
If not, then pass as an optional argument or something

Reply:
I think it would be better if I put it as `SpookNameRate.debug` everywhere?